Sprint-1/3-mandatory-interpret/1-percentage-change.js

```diff
@@ -13,10 +13,46 @@ console.log(`The percentage change is ${percentageChange}`);
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+// There are 5 function calls in this file:
+// line 4: carPrice.replaceAll(",", "")
+// line 4: Number(carPrice.replaceAll(",", ""))
+//line 5: priceAfterOneYear.replaceAll("," "")
+// line 5: Number(priceAfterOneYear.replaceAll("," ""))
+// line 9: console.log(`The percentage change is ${percentageChange}`)
+
+
+
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+/*
+ It's a SyntaxError in line 5: priceAfterOneYear = Number(priceAfterOneYear.replaceAll("," ""));.
+ The error is occurring because there is a missing comma between the two arguments of the replaceAll function here ("," "").
+ To fix this problem, we need to add a comma between the two arguments: replaceAll(",", "").
+ */
+
+
 
 // c) Identify all the lines that are variable reassignment statements
+/*reusing the same variable carPrice but updating its value:
+carPrice = Number(carPrice.replaceAll(",", ""));
+
+reusing the same variable priceAfterOneYear but updating its value:
+priceAfterOneYear = Number(priceAfterOneYear.replaceAll(",", ""));
+*/
+
+
 
 // d) Identify all the lines that are variable declarations
+/*declaring a new variable carPrice:
+let carPrice = "10,000";
+
+declaring a new variable priceAfterOneYear:
+let priceAfterOneYear = "8,543";
+*/
```

Comment:
priceDifference and percentageChange are also variables. Although their values are fixed after assignment due to the use of the const keyword, they are still considered variables in JavaScript.